extensions/PhabBugz/lib/Feed.pm

```diff
@@ -420,7 +420,7 @@ sub process_new_user {
     my $phab_user = Bugzilla::Extension::PhabBugz::User->new($user_data);
 
     if (!$phab_user->bugzilla_id) {
-        DEBUG("SKIPPING: No bugzilla id associated with user");
+        DEBUG("USERS: SKIPPING - No bugzilla id associated with user");
```

Comment:
Is this debugging, or would you want to know in production if a phabricator user had no bmo id?